### PR TITLE
Update Prometheus helmchart dependency

### PR DESCRIPTION
**Pull request by meticulo3366**
…, please see github issue #1038 on prometheus community project, blocker for deploying pulsar on Openshift: https://github.com/prometheus-community/helm-charts/issues/1038

### CODE REVIEW

**Comment by meticulo3366** [comment]
@cdbartholomew add anyone else needed to this for review

**Comment by meticulo3366** [comment]
@zzzming can we merge in?

File: helm-chart-sources/pulsar/values.yaml [hunk 1]
```diff
@@ -327,9 +327,9 @@ image:
     pullPolicy: IfNotPresent
     tag: 1.0.0
   burnell:
-    repository: datastax/burnell
+    repository: meticulo3366/burnell
```
Comment (michaeljmarshall) [inline helm-chart-sources/pulsar/values.yaml:330]: This should still be `datastax/burnell`.

File: helm-chart-sources/pulsar/values.yaml [hunk 2]
```diff
@@ -327,9 +327,9 @@ image:
...
     pullPolicy: Always
-    tag: 1.0.0
+    tag: rootless
```
Comment (michaeljmarshall) [inline helm-chart-sources/pulsar/values.yaml:332]: We'll want to release an official version of the non root datastax burnell image and then use that version here.

File: helm-chart-sources/pulsar/values.yaml [hunk 3]
```diff
@@ -345,7 +345,7 @@ image:
   pulsarHeartbeat:
     repository: datastax/pulsar-heartbeat
     pullPolicy: IfNotPresent
-    tag: 1.0.2 
+    tag: v0.0.6
```
Comment (michaeljmarshall) [inline helm-chart-sources/pulsar/values.yaml:348]: @zzzming - can you confirm the right version here?

**Review by michaeljmarshall — CHANGES_REQUESTED** [review]

File: helm-chart-sources/pulsar/values.yaml (hunk 2 above)
Comment (meticulo3366) [reply helm-chart-sources/pulsar/values.yaml:332]: let me try building

**Comment by lhotari** [comment]
I created #22 with just the kube-prometheus-stack upgrade.

**Comment by lhotari** [comment]
Closing this PR since other smaller PRs cover the changes.

**Pull request closed without merging**